webapp/app.py

```diff
@@ -532,6 +535,26 @@
     r"/security/<regex('(cve-|CVE-)\d{4}-\d{4,7}'):cve_id>", view_func=cve
 )
 
+security_vulnerabilities_path = "/security/vulnerabilities"
```

Comment:
I made a mistake on my rebase and now the feature branch is missing these changes. They were already reviewed and merged here https://github.com/canonical/ubuntu.com/pull/14679